test/test_helper.rb

```diff
@@ -1,6 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'daimon_skycrawlers'
 
+require 'faraday'
```

Comment:
Is this move to `lib/daimon_skycrawlers.rb` ?
And should [this](https://github.com/bm-sms/daimon-skycrawlers/blob/use-stubs-in-crawler-test/lib/daimon_skycrawlers/crawler.rb#L3) be removed?